### ISSUE TITLE
Error regarding Param_local_red_1el

### ISSUE THREAD

**Issue by Rdemeloo1** [genssi-developer/GenSSI#22]
I am running GenSSI2, but when including some parameters for analysis, I run into this error:

**Unrecognized function or variable 'Param_local_red_1el'.

Error in genssiOrderTableau>displayRemainingParameters (line 618)
                    Param_local=[Param_local Param_local_red_1el];

Error in genssiOrderTableau (line 212)
                        displayRemainingParameters...

Error in genssiMain (line 137)
    [options,results] = genssiOrderTableau(model,results,RJacParam01,ECC,rParam,options);**

I checked the referred file, and it is commented as:
![image](https://user-images.githubusercontent.com/123201111/230395724-d66701ff-47fb-4e48-807d-0c26e543b5c6.png)

I am trying to figure out how to solve this issue. Any tips?

Thank you!

**Comment by thomassligon** [comment]
Hi Rebeca,
could you send me a model that provokes the error? You can send it to my email address tom@thomassligon.info and, if it is confidential (for example unpublished research), let me know, so that I don't put it on GitHub.

**Comment by Rdemeloo1** [comment]
I sent it to your email. Thank you!

**Comment by thomassligon** [comment]
Hi Rebeca,
do you have the time and patience to do some testing? I have spent a few hours testing on the basis of the model you sent me, and I haven't yet been able to understand what was intended by these lines of code. In particular, neither Param_local_red_1el nor global_ident_par_red_1el is defined anywhere. I can see that red means reduced (after a parameter has been identified and the list has been reduced) and 1el means one element. I have simply commented out these two lines of code. My code currently looks like this:

                if display_tableau_RJacparam_new_x~=1 % is this if required??!!
%                    Param_local=[Param_local Param_local_red_1el];
%                    global_ident_par=[global_ident_par global_ident_par_red_1el]; 
                    disp('start test issue #22');
                    disp('Param_local:');
                    disp(Param_local);
                    disp('global_ident_par:');
                    disp(global_ident_par);
                    disp('reduced_param:');
                    disp(reduced_param);
                    disp('end test issue #22');
                end

**Comment by Rdemeloo1** [comment]
Hi Thomas,

Thank you for taking the time to help me. I wonder why those two are not defined. Is there someone we can ask about it?

Thanks!

Rebeca


________________________________

**Comment by thomassligon** [comment]
I plan to ask the oigina author Oana. I still have one bug, the second in issue 20.

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by thomassligon** [comment]
I didn't get any feedback on the original intent for these lines, so I just commented them out.

**Comment by Rdemeloo1** [comment]
Thank you for the feedback!

Best,

Rebeca

________________________________

**Comment by Rdemeloo1** [comment]
Hi Thomas,

Is there a way of knowing which parameters are globally identifiable or locally identifiable only by looking at the identifiability tableaus?

I am running into a memory error when I try using a higher number of Lie derivatives, but it still generates the tableaus, so I was wondering if I can use them to check which of the identifiable are local and which are globally identifiable.

Thank you,


Rebeca

________________________________

**Comment by thomassligon** [comment]
Dear Rebeca,

I see that this has been open for some time. Unfortunately, I haven’t made any progress on your request.
On the issue # 22, as I already mentioned, I simply commented out the code so that it wouldn’t cause the error. Then I asked the original author of the code, Oana Chiṣ, to help me, but she hasn’t replied to my email of April 28.
Then on your last question, this looks like a simple question about identifiability tableaus. Since these were invented by Eva Balsa Canto, who also mentored Oana on the first version, I asked her for help, but she hasn’t replied to my email of July 12.

In summary, I am not able to answer all of the questions myself, and none of the co-authors are helping me. Since this is open-source software, created without a separate budget or support agreement, I am pessimistic about our chances to clarify these topics.

Tom
(Dr. Thomas S. Ligon)
***@***.******@***.***>
Frohnloher Str. 6a
81475 Muenchen
Germany
Tel. +49(89)74575075

**Comment by Rdemeloo1** [comment]
Dear Thomas,

I appreciate your feedback. Thank you for the effort and time invested.


Best regards,

Rebeca

________________________________

**Comment by thomassligon** [comment]
Dear Rebeca,
Dear Thomas,
I am very sorry, it seems that I missed your e-mail regarding the tableaus. I try to reply these emails asap, so you can insist next time if I take that long.
With the tableaus you may indeed get visiual information about the identifiability. Taking into consideration that the rows correspond to the generating series coefficients and the columns regard the parameters: if a coefficient depends on one parameter, you will see a black box in the intersection of the corresponding row and column, then the parameter is globally identifiable; if there is at least one black box per parameter (independently of the row) then there is an indication of local identifiability (at least); if you have solved for many Lie Derivatives, and there are full white columns, that may be an indication of lack of structural identifiability (but of course, there is no guarantee as the series would be infinite).

I hope this helps. I'm starting my annual vacation today, but I will be happy to take further questions or try to solve those related to the software once I am back (end of August).

Please accept my apologies again.
With my best regards,
Eva


El Lunes, Julio 31, 2023 09:44 CEST, "Thomas Ligon (Office 365)" ***@***.***> Ha escrito:
